lib/auth.js

```diff
@@ -9,11 +9,24 @@ const self = {}
 
 self.url = null
 self.threshold = 10
-self.scope = 'openid stig-manager:collection stig-manager:stig:read stig-manager:user:read'
 self.key = options.clientKey
 self.authenticateFn = options.clientKey ? authenticateSignedJwt : authenticateClientSecret
 self.authentication = options.clientKey ? 'signed-jwt' : 'client-secret'
 
+const scopePrefix = options.scopePrefix
+
+const scopeArray= [
+  `${scopePrefix}openid`,
```

Comment:
the `openid` scope is unlikely to require a prefix, since it is defined in the standard as a bare scope and is not an API-specific scope